AGENTS.md

```
@@ -0,0 +1 @@
+Use `just` command to interact with this repo: build, test, etc...
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)